dev-python/histoprint/histoprint-2.4.0.ebuild

```diff
@@ -0,0 +1,21 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Pretty print histograms to the console"
+HOMEPAGE="https://github.com/scikit-hep/histoprint"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/click-7.0.0[${PYTHON_USEDEP}]
+	>=dev-python/uhi-0.2.1[${PYTHON_USEDEP}]
+"
+BDEPEND="${RDEPEND}"
```

Comment:
Usually we don't need the rdeps at build time for python packages, unless some building script imports them. Is that the case here?